README.md

```diff
@@ -25,39 +25,43 @@ steps:
 - run: npm test
 ```
 
```

Comment:
Add a note somewhere in this readme to not reference `master` and link to the versioning doc in the toolkit